engine/runtime/src/main/java/org/enso/interpreter/caches/Cache.java

```diff
@@ -218,6 +218,11 @@ public final Optional<T> load(EnsoContext context) {
                   }
 
                   logger.log(logLevel, "Unable to load a cache [" + logName + "]");
+                } catch (IOException e) {
+                  logger.log(
```

Comment:
Twice the same log?

Reply:
My goal was to log just the message with the default log level, but allow logging of the whole stacktrace when somebody increases the log level. What's the right way to do it?

Reply:
I would say that whether to include or not the stacktrace is the role of the logging configuration.
See the difference in `%nopex` in https://github.com/enso-org/enso/blob/develop/engine/launcher/src/main/resources/application.conf#L30 configuration - no stacktraces in console but full stacktrace in the log file.

Reply:
Is this what you want me to do? 86ebf06

Reply:
> no stacktraces in console but full stacktrace in the log file.

I believe the https://github.com/enso-org/enso/issues/8421#issuecomment-1876296226 says the stacktrace was visible in the console as well.